core/src/broadcast_stage/slot_transmit_shreds_cache.rs

```diff
@@ -0,0 +1,675 @@
+use super::*;
+use solana_ledger::shred::Shred;
```

Comment:
Is there any justification for having this extra caching thing? Did we try just loading from blockstore and it was too slow?